EnemyCollision.cs

```diff
@@ -14,7 +14,8 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        
+        Movement();
```

Comment:
Good job. This is how we call a function! Remember that since Movement()  and Shoot() are called in the Update() function, that means it will be constantly called - once per frame.